dsgrid/utils/spark.py

```diff
@@ -63,6 +63,12 @@ def read_dataframe(filename, cache=False, require_unique=None, read_with_spark=T
     Supported formats when read_with_spark=True: .csv, .json, .parquet
     Supported formats when read_with_spark=False: .csv, .json
 
+    When reading CSV files on AWS read_with_spark should be set to False because the
+    files would need to be present on local storage for all workers. The master node
+    will sync the config files from S3, read them with standard filesystem system calls,
+    and then convert the data to Spark dataframes. This could change if we ever decide
+    to read CSV files with Spark directly from S3.
```

Comment:
This might be the answer to one of my questions above, but I'll let you confirm @daniel-thom.

Reply:
Yes.